src/annotations/core/AnnotationArea.cpp

```diff
@@ -81,9 +81,9 @@ void AnnotationArea::insertImageItem(const QPointF &position, const QPixmap &ima
 
 void AnnotationArea::replaceBackgroundImage(const QPixmap &image)
 {
-	delete mImage;
-	mImage = addPixmap(image);
-	setSceneRect(image.rect());
+	delete mImage;    
+    mImage = addPixmap(image);
+    setSceneRect(ScaledSizeProvider::getUnscaledRect(image.rect(), image.devicePixelRatio()));
```

Comment:
I would expect here something like `ScaledSizeProvider::getUnscaledRect(image.rect())` the scale factor is already provided by the methods in the class `getXScaleFactor()` and `getYScaleFactor()`.